app/src/main/java/org/mozilla/gecko/search/SearchWidgetProvider.kt

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.gecko.search
+
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import mozilla.components.feature.search.widget.AppSearchWidgetProvider
+import mozilla.components.feature.search.widget.BaseVoiceSearchActivity
+import mozilla.components.feature.search.widget.SearchWidgetConfig
+import mozilla.components.support.utils.PendingIntentUtils
+import org.mozilla.focus.R
+import org.mozilla.focus.activity.IntentReceiverActivity
+import org.mozilla.focus.ext.components
+import org.mozilla.focus.searchwidget.VoiceSearchActivity
+
+class SearchWidgetProvider : AppSearchWidgetProvider() {
+
+    // Implementation note:
```

Comment:
This note does not apply to Focus.

Reply:
Also about this. We don't need the gecko package for Focus. That was a requirement just for supporting the Fennec -> Fenix migration.

Reply:
Done